pymc_experimental/distributions/__init__.py

```diff
@@ -18,7 +18,9 @@
 """
 
 from pymc_experimental.distributions.continuous import GenExtreme
+from pymc_experimental.distributions.multivatiate import R2D2M2CP
 
 __all__ = [
     "GenExtreme",
+    "R2D2M2CP",
```

Comment:
Don't forget to add it to the docs

Reply:
I also added histogram_approximation to this scope as it was like this in docs